README.md

```diff
@@ -165,14 +165,15 @@ project.pbxproj.objects.addObject(homeGroup)
 
 ```swift
 let homeGroup = PBXGroup(reference: "xxx", children: [], sourceTree: .group, path: "Home")
-let homeViewController = PBXFileReference(reference: "xxx", sourceTree: .group, path: "HomeViewController.swift")
+let homeViewController = PBXFileReference(reference: project.pbxproj.generateUUID(for: PBXFileReference.self), sourceTree: .group, name: "HomeViewController.swift", path: "HomeViewController.swift")
 homeGroup.children.append(homeViewController.reference)
+pbxproj.objects.addObject(homeViewController)
```

Comment:
Without this file will be not properly added and build file will point to a dangling reference.

Reply:
Good catch!